identity_core/Cargo.toml

```diff
@@ -21,6 +21,7 @@ anyhow = "1.0"
 serde = {version = "1.0", features = ["derive"]}
 serde_json = "1.0"
 serde_cbor = "0.11"
+serde-diff = "0.3.1"
```

Comment:
The output of this is quite large...but I noticed you mentioned on discord that you were looking into some changes so 👍

Reply:
The output is large if the json is not minified.  When its minified, its actually about the same as what I've been working on.  That said, I still want to have a bit more control over the diffing logic, so I will go through with my library and we can discuss it as a group this Wednesday.